lib/publish.js

```diff
@@ -11,7 +11,10 @@ module.exports = async ({npmPublish, tarballDir, pkgRoot}, {publishConfig, name}
 
   if (tarballDir) {
     logger.log('Creating npm package version %s', version);
-    const tarball = await execa.stdout('npm', ['pack', `./${basePath}`]);
+    const tarball = (await execa.stdout('npm', ['pack', `./${basePath}`]))
+      .split('\n')
+      .slice(-1)
```

Comment:
Why using `slice(-1)` and `pop()` here? They seems to do the same thing.
I think `slice(-1)` can be removed, so we keep only `pop()`.